packages/graalvm-wasm/build.gradle.kts

```diff
@@ -11,34 +11,50 @@
  * License for the specific language governing permissions and limitations under the License.
  */
 
-@file:Suppress(
-  "UnstableApiUsage",
-  "unused",
-  "DSL_SCOPE_VIOLATION",
-)
-
-import ElidePackages.elidePackage
+import elide.internal.conventions.elide
+import elide.internal.conventions.publishing.publish
+import elide.internal.conventions.native.NativeTarget
+import elide.internal.conventions.kotlin.KotlinTarget
 
 plugins {
-  id("io.micronaut.library")
-  id("io.micronaut.graalvm")
+  alias(libs.plugins.micronaut.library)
+  alias(libs.plugins.micronaut.graalvm)
 
+  kotlin("jvm")
   kotlin("kapt")
   kotlin("plugin.allopen")
 
-  id("dev.elide.build.native.lib")
-  id("dev.elide.build.publishable")
+  id("elide.internal.conventions")
 }
 
-group = "dev.elide"
-version = rootProject.version as String
+elide {
+  publishing {
+     id = "graalvm-wasm"
+    name = "Elide WASM for GraalWasm"
+    description = "Integration package with GraalVM WASM and Elide."
 
-val encloseSdk = false
+    publish("jvm") {
+      from(components["kotlin"])
+    }
+  }
+  
+  kotlin {
+    target = KotlinTarget.JVM
+    explicitApi = true
+  }
```

Comment:
i believe this targets WASM only...

Reply:
the original build script applied the `dev.elide.build.native.lib` convention, which applies the Kotlin/JVM plugin (indirectly I think), and in general the build was being configured as JVM only (e.g. no Multiplatform targets declared, use of top-level `dependencies` block)

so I can't really see how it was targeting WASM? I thought it's meant to provide GraalVM's WASM engine for the runtime.

Reply:
you are totally right, i was lost (thinking i was in the `wasm` package)